bus.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="bus.css"/>
+</head>
+<body>
+<h1>Bus Mall</h1>
+<div class='app'>
+<figure>
+  <ul>
+  <li id="img-1"></li>
+  <li id="img-2"></li>
+  <li id="img-3"></li>
```

Comment:
Consider creating the three image tags that are needed in the html file and assign or update the src attribute each time you need to change the displayed image.